sceptre/plan/actions.py

```diff
@@ -1069,7 +1069,7 @@ def _wait_for_drift_status(self, detection_id: str) -> dict:
             if elapsed >= timeout:
                 raise TimeoutError(f"Timed out after {elapsed} seconds")
 
-            self.logger.debug(f"{self.stack.name} - Waiting for drift detection")
+            self.logger.info(f"{self.stack.name} - Waiting for drift detection")
```

Comment:
If you're going to log at the info level, I'd suggest a 10 second sleep interval rather than 5.

Reply:
ok